exchanges/opendax/README.md

```diff
@@ -0,0 +1,30 @@
+
+# GoCryptoTrader Opendax Exchange Wrapper
+
+<img src="https://github.com/thrasher-corp/gocryptotrader/blob/master/web/src/assets/page-logo.png?raw=true" width="350px" height="350px" hspace="70">
```

Comment:
make a different README than the template, and commit in irix repository

Reply:
So, I need to move readme file in irix repo?